src/screens/TokenSwap.js

```diff
@@ -425,10 +426,10 @@ const styles = StyleSheet.create({
     // For IOS
     shadowColor: COLORS.lightShadow,
     shadowOffset: { width: 0, height: 2 },
-    shadowOpacity: 0.5,
+    shadowOpacity: 0.1,
```

Comment:
question(non-blocking): Would it be better to use `textColorShadowLighter` instead of applying a color and an opacity? It was designed only with text color in mind, but maybe it's useful here as well.

https://github.com/HathorNetwork/hathor-wallet-mobile/blob/87b5884eac4acb8abc8cd192055323f11c47f03f/src/styles/themes.js#L42-L47